README.md

```diff
@@ -4,6 +4,10 @@
 
 Common SELinux package used across the container ecosystem.
 
+## Usage
+
+When compiling consumers of this project, the `selinux` build tag must be used to enable selinux functionality.
+
 Please see the [godoc](https://godoc.org/github.com/opencontainers/selinux) for more information.
```

Comment:
Now this "more information" refers to using the selinux tag, rather than the overall usage.

I suggest to replace it with something like

```
For complete documentation, see [godoc](https://godoc.org/github.com/opencontainers/selinux).
```

Reply:
Done.